deepcell/data/tracking.py

```diff
@@ -114,13 +399,17 @@ def random_translate(X, y, range=512):
     return X, y
 
 
-def prepare_dataset(track_info, batch_size=32, buffer_size=256,
+def prepare_dataset(tracked_data=None, path=None,
```

Comment:
this will throw an error if its passed `.trk` instead of `.trks` but I don't see a check/warning here or in `Track`

Reply:
Yeah thats a good idea, we should probably check the filepath or the tracked_data shape in `Track`.